packages/chat/tests/Unit/Middleware/Tools/ToolExecutionMiddlewareTest.php

```
@@ -81,7 +81,7 @@ public function testProcessWithRegularResponse(): void
             Usage::empty(),
         );
 
-        $next = fn (AIChatRequest $request, ?AIChatAdapterInterface $adapter) => $response;
+        $next = static fn (AIChatRequest $request, ?AIChatAdapterInterface $adapter) => $response;
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)